routers/user/home.go

```diff
@@ -465,10 +465,44 @@ func Issues(ctx *context.Context) {
 		opts.MentionedID = ctxUser.ID
 	}
 
-	counts, err := models.CountIssuesByRepo(opts)
-	if err != nil {
-		ctx.ServerError("CountIssuesByRepo", err)
-		return
+	var forceEmpty bool
+	var issueIDsFromSearch []int64
+	var keyword string
+	if !isPullList {
+		keyword = strings.Trim(ctx.Query("q"), " ")
+		if bytes.Contains([]byte(keyword), []byte{0x00}) {
```

Comment:
Go supports the null character inside strings, so I don't see how this could be a problem.

Reply:
Thanks, I removed the check. I had copied and pasted this from the issue search code inside a specific repository